src/app/shared/components/template/services/template-field.service.ts

```diff
@@ -14,16 +15,34 @@ export class TemplateFieldService extends AsyncServiceBase {
   constructor(
     private localStorageService: LocalStorageService,
     private dbService: DbService,
-    private translateService: TemplateTranslateService
+    private translateService: TemplateTranslateService,
+    private templateActionRegistry: TemplateActionRegistry
   ) {
     super("TemplateField");
     this.registerInitFunction(this.initialise);
+    this.registerTemplateActionHandlers();
   }
 
   private async initialise() {
     await this.ensureAsyncServicesReady([this.dbService, this.translateService]);
     this.ensureSyncServicesReady([this.localStorageService]);
   }
+
+  private registerTemplateActionHandlers() {
```

Comment:
It wasn't strictly necessary to move the field actions to their parent service (and use the new registry system to register), as I later realised I could simply override all global action registry entries which take precedence over anything defined in the template-action service. However now it's here it feels to me to be a more sensible place to be and helps tidy the template-action registry code a little